src/api/index.ts

```
@@ -52,11 +53,24 @@ export const instance = async (
+      status: context.status,
+    });
+    if ((err as Error).name === 'TimeoutError') {
+      sentry.captureException(new ServerNotRespondingError());
       throw new ServerNotRespondingError();
-    else {
-      if (!error || !(error && error[`${(err as Response).status}`]))
-        throw new Error(`서버에서 오류가 발생했습니다. (${err.name})`);
+    } else {
+      if (!error || !(error && error[`${(err as Response).status}`])) {
+        const serverError = new Error(
+          `서버에서 예기치 않은 오류가 발생했습니다. (${err.name})`,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)